src/components/Audioplayer/ScrollButton/index.js

```diff
@@ -1,9 +1,19 @@
 import React, { PropTypes } from 'react';
+import styled from 'styled-components';
 import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
 import Tooltip from 'react-bootstrap/lib/Tooltip';
 import LocaleFormattedMessage from 'components/LocaleFormattedMessage';
 
-const style = require('../style.scss');
+const ScrollLink = styled.a`
+  width: 100%;
+  display: inline-block;
+  cursor: pointer;
+  padding-right: 1.5%;
+  color: ${props => (props.active ? props.theme.brandPrimary : props.theme.textColor)};
+  outline: none;
+  cursor: pointer;
+  margin-bottom: 0;
+`;
```

Comment:
I definitely understand the questions @naveed-ahmad raises. 
I'd recommend you watch/skip through this video: https://pusher.com/sessions/meetup/reactivate-london/from-css-preprocessors-to-styled-components // https://www.smashingmagazine.com/2017/01/styled-components-enforcing-best-practices-component-based-systems/

- Code elimination with CSS/scss is impossible so this will be the biggest win.
- Maintainable code and style and no context switching
- Removing bootstrap

I agreed it is a bit awkward and I don't agree with styled-components fully, however I think (and it took me a while to see) it is the correct way to react components. 

sometimes I wish we could do:

```
const style = X`
color: yellow;
`
<div style={style} /> 

```

rather than using `Styled.div`

It really lends it self to component libs, it will take a while to fully migrate everything across. 

I agreed with all the points @mmahalwy mentioned.